PerDiem/PerDiem/models/PerDiem.m

```diff
@@ -56,7 +56,7 @@ + (void)perDiemsForPeriod:(DTTimePeriod *)period
             CGFloat totalSpentSoFar = 0;
             CGFloat spentOnThatDay = 0;
             for (Transaction *transaction in transactions) {
-                if ([transaction.transactionDate earlierDate:day]) {
+                if ([transaction.transactionDate earlierDate:day] || [self souldUsePendingTransaction: transaction day: day]) {
```

Comment:
I haven't messed with the date stuff much, but doesn't [earlierDate](https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Classes/NSDate_Class/index.html#//apple_ref/occ/instm/NSDate/earlierDate:) / laterDate return a date value, not a boolean?